.github/workflows/test-code.yml

```diff
@@ -33,6 +33,21 @@ jobs:
       - name: "Run quick linters"
         run: |
           ./gradlew -PGITHUB_ACCESS_TOKEN=${{ secrets.ACCESS_TOKEN_GRADLE_BUILD }} spotlessCheck shellcheck lintDockerfile
+
+      - name: "Python isort"
+        uses: isort/isort-action@master
+        with:
+          requirementsFiles: "domain-cc/python_src/requirements.txt \
+                              domain-cc/python_src/dev-requirements.txt \
+                              domain-rrd/service-python/requirements.txt \
+                              domain-rrd/service-python/pdfgenerator/src/requirements.txt \
+                              domain-rrd/service-python/featuretoggle/src/requirements.txt \
+                              domain-rrd/service-python/assessclaimdc7101/src/requirements.txt \
+                              domain-rrd/service-python/assessclaimdc6602v2/src/requirements.txt \
+                              domain-rrd/service-python/assessclaimdc6602/src/requirements.txt \
+                              domain-rrd/service-python/assessclaimdc6522/src/requirements.txt \
+                              domain-rrd/service-python/assessclaimcancer/src/requirements.txt"
```

Comment:
Is there a way to use a regex like `**/requirements.txt` so that this is less fragile? If not, consider adding an action to find the files and set a variable that this action uses.